components/feature/prompts/src/main/java/mozilla/components/feature/prompts/creditcard/CreditCardPicker.kt

```diff
@@ -92,6 +93,9 @@ class CreditCardPicker(
         try {
             if (promptRequest != null) {
                 promptRequest.onDismiss()
+                sessionId?.let {
```

Comment:
@gabrielluong are you familiar why we were not consuming the prompt request here and returning early?

Reply:
More context can be found here https://github.com/mozilla-mobile/android-components/pull/12142#discussion_r873096373